src/srcShared/fs.ts

```diff
@@ -159,3 +179,5 @@ export class FileSystemCommon {
         return vscode.Uri.file(path)
     }
 }
+
+export const fsCommon = FileSystemCommon.instance
```

Comment:
Yay :) Could even just name it `fs`, since it will be rare to use multiple "fs" modules at once.
```suggestion
export const fs = FileSystemCommon.instance
```